src/components/PastAnal/CandidateSummary.vue

```diff
@@ -0,0 +1,140 @@
+<style lang="scss">
+@import '@/styles/main.scss';
+.voteSum__card {
+  width: 100%;
+  height: 100%;
+  min-height: 114px;
+  @include media-breakpoint-up(md) {
+    min-height: 244px;
+  }
+}
```

Comment:
1. 這裡的 `.voteSum__card` class 跟 `VoteSummaryCard.vue` 元件的 `.voteSum__card` class 名稱重複，且沒有用到，故可以刪除。
2. "<style lang="scss" `scoped`>" 這裡最好加個 `scoped`，可以避免污染到其他元件有用到相同名稱的 class。

```suggestion
<style lang="scss" scoped>
@import '@/styles/main.scss';
```